src/Codeception/Configuration.php

```diff
@@ -500,6 +500,126 @@ public static function projectDir()
     }
 
     /**
+     * Returns path to a given directory relative to $projDir.
+     * @param string $path
+     * @param string $projDir
+     * @param string $dirSep
+     * @return string
+     */
+    public static function getRelativeDir($path, $projDir, $dirSep = DIRECTORY_SEPARATOR)
+    {
+        // ensure $projDir ends with a trailing $dirSep
+        $projDir = preg_replace('/'.preg_quote($dirSep, '/').'*$/', $dirSep, $projDir);
+        // if $path is a within $projDir
+        if (self::fsCaseStrCmp(substr($path, 0, strlen($projDir)), $projDir, $dirSep) == 0) {
+            // simply chop it off the front
+            return substr($path, strlen($projDir));
+        }
+        // Identify any absoluteness prefix (like '/' in Unix or "C:\\" in Windows)
+        $pathAbsPrefix = self::getPathAbsolutenessPrefix($path, $dirSep);
+        $projDirAbsPrefix = self::getPathAbsolutenessPrefix($projDir, $dirSep);
+        $sameAbsoluteness = (self::fsCaseStrCmp($pathAbsPrefix['wholePrefix'], $projDirAbsPrefix['wholePrefix'], $dirSep) == 0);
+        if (!$sameAbsoluteness) {
+            // if the $projDir and $path aren't relative to the same
+            // thing, we can't make a relative path.
+
+            // if we're relative to the same device ...
+            if (strlen($pathAbsPrefix['devicePrefix']) &&
+                (self::fsCaseStrCmp($pathAbsPrefix['devicePrefix'], $projDirAbsPrefix['devicePrefix'], $dirSep) == 0)
+            ) {
+                // ... shave that off
+                return substr($path, strlen($pathAbsPrefix['devicePrefix']));
+            }
+            // Return the input unaltered
+            return $path;
+        }
+        // peel off optional absoluteness prefixes and convert
+        // $path and $projDir to an subdirectory path array
+        $relPathParts = array_filter(explode($dirSep, substr($path, strlen($pathAbsPrefix['wholePrefix']))), 'strlen');
```

Comment:
'strlen' is not necessary, array_filter removes empty strings when callback is not provided.